hyperion/renderer/index.js

```diff
@@ -189,6 +189,8 @@ const renderer = (req: express$Request, res: express$Response) => {
       );
     })
     .catch(err => {
+      // Avoid memory leaks, see https://github.com/styled-components/styled-components/issues/1624#issuecomment-425382979
+      sheet.complete();
```

Comment:
This api has changed.

Currently is `sheet.seal()` instead of `sheet.complete()`

Also would be even better if you call it on the `finally` block instead of catch. :)

Reply:
We're on v3, so it's `.complete` right?

Reply:
I'm 100% sure when the API changed, we are on v6 so it might be that you need to call `.complete` instead